src/tests/ftest/server/metadata_svc_ops.py

```diff
@@ -0,0 +1,141 @@
+            workload_cycles (int): Number of metadata workload test cycles per test loop.
+                Defaults to 5000.
+            test_loops (int): Number of metadata workload test loops. Defaults to 1.
+
+        Returns:
+            test_time: List of time consumed per test loops of metadata workload test cycles.
+
+        """
+        test_time = []
+        try:
+            container = self.get_container(pool, create=False)
+            if container.daos:
+                container.daos.verbose = False
+            container.create()
+            self.log.info("Successfully created #%s container", num_of_cont)
+        except (DaosTestError, TestFail) as err:
+            self.fail(
+                "#({}.{}) container create failed. err={}".format(pool.label, num_of_cont, err))
+        for ind in range(test_loops):
+            start = time.time()
+            for _ in range(workload_cycles):
+                container.close()
+                container.open()
+            elapsed_time = time.time() - start
+            self.log.info("Completed container Metadata test-loop: %d, elapsed_time: %f",
+                          ind + 1, elapsed_time)
+            test_time.append(elapsed_time)
+        for ind in range(test_loops):
+            self.log.info("Test time of Metadata test-loop: %d,  %f",
+                          ind, test_time[ind])
+        container.close()
+        return test_time
+
+    def test_metadata_dup_rpc(self):
+        """JIRA ID: DAOS-15937 metadata duplicate rpc detection time consuming.
+
+        Test Steps:
+            1. Bring up DAOS server.
+            2. Create pool1 with specified property svc_ops_entry_age.
+            3. Create containers by ThreadManager.
+            4. Run specified metadata workload cycles in multiple test loops (N cycles per loop).
+            5. Create pool2 with property svc_ops_enable:0.
+            6. Create containers by ThreadManager on pool2.
+            7. To establish a "baseline" time (without duplicate rpc detection), perform test
+               step 4 on pool2, calculating average time per loop executed.
+            8. Compare all metadata workload times (with duplicate rpc detection) to the average
+               baseline time (without duplicate rpc).
+
+        :avocado: tags=all,full_regression
+        :avocado: tags=hw,medium,md_on_ssd
+        :avocado: tags=server,metadata
+        :avocado: tags=DuplicateRpcDetection,test_metadata_dup_rpc
+        """
+        num_of_cont = self.params.get("number_thread", '/run/metadata/*', default=1)
+        w_cycles = self.params.get("workload_test_cycles", '/run/metadata/*', default=5000)
+        t_loops = self.params.get("test_loops", '/run/metadata/*', default=10)
+        threshold_factor = self.params.get("threshold_factor", '/run/metadata/*', default=1.75)
+
+        self.log_step("Create pool with properties svc_ops_entry_age.")
+        pool1 = self.get_pool(dmg=self.dmg.copy())
```

Comment:
We can just get the command here
```suggestion
        pool1 = self.get_pool(dmg=self.get_dmg_command().copy())
```